src/robusta/cli/self_host.py

```diff
@@ -0,0 +1,180 @@
+import secrets
+import string
+import typer
+import yaml
+import json
+from typing import Any
+from pydantic import BaseModel
+from .backend_profile import BackendProfile
+import jwt as JWT
+
+
+def gen_secret(length: int) -> str:
+    return "".join(
+        secrets.choice(string.ascii_letters + string.digits) for _ in range(length)
+    )
+
+
+def write_values_files(
+    values_path: str,
+    backendconfig_path: str,
+    values: dict[str, Any],
+    backendProfile: BackendProfile,
+):
+    with open(values_path, "w") as output_file:
+        yaml.safe_dump(values, output_file, sort_keys=False)
+        typer.secho(
+            f"Saved configuration to {values_path} - save this file for future use!",
+            fg="red",
+        )
+
+    with open(backendconfig_path, "w") as output_file:
+        json.dump(
+            backendProfile.dict(exclude={"custom_profile"}), output_file, indent=1
+        )
+        typer.secho(
+            f"Saved configuration to {backendconfig_path} - save this file for future use!",
+            fg="red",
+        )
+
+
+class RobustaUI(BaseModel):
+    RELAY_HOST: str = ""
```

Comment:
Why do we need the default value "" here?

Reply:
removed here